spec/helpers.lua

```diff
@@ -0,0 +1,43 @@
+-- Pre-load the vips module
+require "vips"
+
+local assert = require "luassert.assert"
+local say = require "say"
+
+-- TODO: Wait for https://github.com/Olivine-Labs/luassert/issues/148
+-- Meanwhile, we're applying patch #150 here.
+
+-- Pre-load the ffi module, such that it becomes part of the environment
+-- and Busted will not try to GC and reload it. The ffi is not suited
+-- for that and will occasionally segfault if done so.
+local ffi = require "ffi"
```

Comment:
I like this fix for busted, nice! I never figured this one out.

Reply:
Credits to: https://github.com/Olivine-Labs/luassert/pull/150